vehicle/raw_vehicle_cmd_converter/src/accel_map.cpp

```diff
@@ -82,30 +68,17 @@ bool AccelMap::getThrottle(double acc, double vel, double & throttle)
 bool AccelMap::getAcceleration(double throttle, double vel, double & acc)
```

Comment:
Adding const to the arguments of getAcceleration, and defining `calmped_vel/acc/throttle` with const would be better since we can add const to all the variables.

Reply:
Same to other functions like getThrottle.